src/lib/storage/refinements/partition_attributes.rb

```diff
@@ -42,9 +42,9 @@ def label
           end
 
           # @return [DiskSize]
-          def size
-            DiskSize.KiB(size_k)
-          end
+#          def size
+#            DiskSize.KiB(size_k)
+#          end
 
```

Comment:
This should go away completely. size is provided in libstorage. There might be some size_k's left to kill, though. But let's do this later.